ManageIQ/manageiq
# Openstack infra get missing network ports from smartstate

OSP-d deployments usually use multiple networks(internal_api, storage, external, etc) and we should be able to show these associated to the nodes. Currently the nodes only show the ip address from the clplane network. We can get the networks and ports from the undercloud neutron and  correlate them with the data passed in the network environment files from Heat.

Links
----------------

* https://bugzilla.redhat.com/show_bug.cgi?id=1266152
* WIP PR before Network provider was introduced https://github.com/ManageIQ/manageiq/pull/4829

Steps for Testing/QA
-------------------------------

* setup openstack infra provider
* run refresh, SSA
* open Networks - Network Port and you should see Port belonging to the infra provider
